cadl-extension/src/code-model-builder.ts

```diff
@@ -285,6 +285,10 @@ export class CodeModelBuilder {
               securitySchemes.push(keyScheme);
             }
             break;
+          case "http":
+          {
+            this.logWarning(scheme.scheme + " auth method is currently not supported.");
+          }
```

Comment:
do we need to add `break;` at the end?

Reply:
Added `break` for future proofing this when we add more cases below `http`.